aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/Translator.java

```diff
@@ -261,6 +267,71 @@ public static CreateDbInstanceRequest createDbInstanceRequest(
                 .build();
     }
 
+    static Instant stringToInstant(String restoreTimeString) {
+        if (com.amazonaws.util.StringUtils.hasValue(restoreTimeString)) {
```

Comment:
Let's please unify our imports: we already have `software.amazon.awssdk.utils.StringUtils`. I'm quite sure we have an equivalent of `hasValue` there.